pages/common/drupal.md

```diff
@@ -0,0 +1,21 @@
+# drupal
+
+> Drupal Console.
+> The Drupal CLI. A tool to generate boilerplate code, interact with and debug Drupal.
+> More information: <https://drupalconsole.com/>.
+
+- Install module "foo":
+
+`drupal module:install {{foo}}`
+
+- Uninstall module "foo":
+
+`drupal module:uninstall {{foo}}`
+
+- Clear all caches:
+
+`drupal cache:rebuild`
+
+- View current Drupal Installation status:
```

Comment:
Should `**I**nstallation` be **i**nstallation here?

Reply:
Yes, I think so.